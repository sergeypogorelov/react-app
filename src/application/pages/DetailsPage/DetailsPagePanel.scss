@import '../../common/scss/base.scss';

.info {
    padding: 15px 30px;
    background: $infoBgColor;
}

.info-content, .info-sort {
    font-size: 18px;
    font-weight: bold;
    color: $infoDefaultColor;
}

.info-content {
    float: left;
}

.info-sort {
    float: right;
}

.info-sort-link {
    margin-left: 15px;
    color: $infoPrimaryColor;
}

.info-sort-link.active {
    color: $infoDefaultColor;
}