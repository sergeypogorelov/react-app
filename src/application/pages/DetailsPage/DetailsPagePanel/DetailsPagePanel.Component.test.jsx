import React from 'react';
import { HashRouter as Router } from 'react-router-dom';

import renderer from 'react-test-renderer';

import DetailsPagePanel from './DetailsPagePanel.Component';

test('Component DetailsPagePanel works fine.', () => {
    
    const props = {
        filmLoaded: true,
        film: {
            director: 'John Smith'
        }
    };

    let component = renderer.create(
        <DetailsPagePanel { ...props } />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    props.filmLoaded = false;

    component = renderer.create(
        <DetailsPagePanel { ...props } />
    );

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});