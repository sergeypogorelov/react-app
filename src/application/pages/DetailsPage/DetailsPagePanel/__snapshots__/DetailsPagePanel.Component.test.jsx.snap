// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component DetailsPagePanel works fine. 1`] = `
<div
  className="info clearfix"
>
  <div
    className="info-content"
  >
    Films by 
    John Smith
  </div>
</div>
`;

exports[`Component DetailsPagePanel works fine. 2`] = `
<div
  className="info"
/>
`;
